3-enrich/scala-common-enrich/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/enrichments/registry/pii/package.scala

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright (c) 2017-2018 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+package com.snowplowanalytics.snowplow.enrich.common
+package enrichments.registry
+
+// Scala libraries
+import com.fasterxml.jackson.databind.{ObjectMapper, SerializationFeature}
+import com.jayway.jsonpath.spi.json.JacksonJsonNodeJsonProvider
+import com.jayway.jsonpath.{Configuration, Option => JOption}
+
+// Snowplow
+import outputs.EnrichedEvent
+
+package object pii {
+  type ModifiedFields  = List[ModifiedField]
+  type ApplyStrategyFn = (String, PiiStrategy) => (String, ModifiedFields)
+  type MutatorFn       = (EnrichedEvent, PiiStrategy, ApplyStrategyFn) => ModifiedFields
+
+  val JsonMutators   = Mutators.JsonMutators
+  val ScalarMutators = Mutators.ScalarMutators
+
+  // Configuration for JsonPath
+  private[pii] val JacksonNodeJsonObjectMapper = {
+    val objectMapper = new ObjectMapper()
+    objectMapper.configure(SerializationFeature.FAIL_ON_EMPTY_BEANS, false)
```

Comment:
can we comment why this configuration was chosen, same below with `SUPPRESS_EXCEPTIONS`